doc/snmp/snmp-configdb-migration-hld.md

```diff
@@ -54,18 +55,18 @@ admin@switch1:~$ redis-cli -n 4 hgetall "SNMP_COMMUNITY|Jack"
 SNMP_USER|<user>
 
 admin@switch1:~$ redis-cli -n 4 hgetall "SNMP_USER|Travis"
- 1) "SNMP_USER_ENCRYPTION_TYPE"
- 2) "AES"
- 3) "SNMP_USER_AUTH_TYPE"
- 4) "SHA"
- 5) "SNMP_USER_ENCRYPTION_PASSWORD"
- 6) "TravisEncryptPass"
+ 1) "SNMP_USER_TYPE"
+ 2) "Priv"
+ 3) "SNMP_USER_PERMISSION"
```

Comment:
The order of fields does not matter. Seem there is no change, could you keep old version?

Reply:
I'd prefer to have this match up with what we get in the redis db output that is now the finalized version.  To me that makes more sense.